utils/build/docker/php/common/php.ini

```diff
@@ -35,3 +35,4 @@ datadog.experimental_api_security_enabled=1
 datadog.api_security_request_sample_rate=1
 
 datadog.trace.log_file=/var/log/system-tests/tracer.log
+datadog.trace.log_level=trace
```

Comment:
Could you have a quick look to https://github.com/DataDog/system-tests/blob/main/docs/execute/dd-trace-debug.md :) 

Though, nothing is lost, the `--force-dd-trace-debug` should help you. We can also imagine to extend it it you need `trace` level

Reply:
Hmm, it looks like that's only for local runs? I've been trying to reproduce the nightly behavior locally, but haven't had any luck (lowering timeouts, reducing allotted CPUs, etc.) What should we do for this case?

Reply:
No, you can use it in your CI, it should work the same. Just, it must not be the default.

It's for the error on debugging scenarios ?